docs/source/platform/schema-registry.md

```diff
@@ -122,14 +122,56 @@ jobs:
       # commands against it
       - run: sleep 5
 
-      # This will authenticate using the `ENGINE_API_KEY` environment
-      # variable. If the GraphQL server is available elsewhere than
-      # http://localhost:4000/graphql, set it with `--endpoint=<URL>`.
-
-      # When running on the 'master' branch, publish the latest version
+      # When running on the 'master' branch, push the latest version
       # of the schema to Apollo Engine.
       - run: |
           if [ "${CIRCLE_BRANCH}" == "master" ]; then
             apollo service:push --tag=master
           fi
 ```
+
+<h2 id="history">Viewing schema change history</h2>
+
+Once you have uploaded your schema, you can view it through [Apollo's UI in Engine](https://engine.apollographql.com) by browsing to the History page in your Service. The published schema will appear in your history log along with a list of changes comparing it to the previously published versions of your schema. You can view the full contents of the most recently published version of your schema in the Explorer page.
+
+<img src="../images/schema-history.png" width="100%" alt="Schema history page in the Engine UI">
+
+> **Note:** in addition to pushing on deploy, registering the schema on the `master` branch can provide a granular history of schema changes. This is often useful for teams without continuous delivery
+
+<h2 id="schema-tags">Managing environments</h2>
+
+Product cycles move fast, and it's common for a schemas to be slightly different across environments as changes make their way through your system. To accommodate for this, the schema registry allows each schema to be registered under a "schema tag". Tags are mostly commonly used to represent environments, but can also be used to represent things like branches and future schemas.
```

Comment:
The last sentence confuses me because I am confused by the overloaded term and don't understand the utility of a schema tag from this paragraph. I suggest we remove `but can also be used to represent things like branches and future schemas.`, since a tag as a branch will not be a part of the new schema validation snapshot